bl_utils.h

```diff
@@ -0,0 +1,12 @@
+#ifndef BL_UTILS_H_INCLUDED
+#define BL_UTILS_H_INCLUDED
+
+#define BL_WD_TO_MAX_MS         (8000)
+#define UART_READABLE_MAX_US    (1000000)
```

Comment:
this should be in config header, and also we should name it to something related to the context, here we wait for stay in BL trigger so :
`BL_STAY_IN_BL_UART_TRIGGER_TO_MAX_US`

Reply:
UART_WAIT_FOR_TRIGGER_TOUT_US